src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -285,6 +290,7 @@ int run(String[] args) {
             if (DEBUG) {
                 e.printStackTrace(log);
             }
+            cleanupOutput(outputPath);
             return EXIT_ERROR;
         } catch (BadArgs e) {
```

Comment:
Can you confirm that this change is benign for the case that the output directory exists? I believe it's a BadArgs case so it won't attempt to delete a file or tree if it exists but I'd like confirmation.

Reply:
Will check that.